src/app/movie/MovieBanner.tsx

```diff
@@ -1,147 +1,113 @@
 import React from 'react';
 
 import {
+  Badge,
   Box,
   Button,
   Center,
+  CircularProgress,
   Divider,
   Flex,
   Heading,
   Image,
   Text,
-  Badge
 } from '@chakra-ui/react';
+import dayjs from 'dayjs';
 
-import { useMoviesList } from './movies.service';
-
-export const MovieBanner = ({ movieId }: { movieId: number}) => {
-  let display: any = 'Just preventing error by this char';
-
-  const { movies, isLoadingPage } = useMoviesList();
-
-  const setMonth = (month: number) => {
-    switch (month) {
-      case 1:
-        return 'january';
-
-      case 2:
-        return 'February';
-
-      case 3:
-        return 'March';
-
-      case 4:
-        return 'April';
-
-      case 5:
-        return 'May';
-
-      case 6:
-        return 'June';
-
-      case 7:
-        return 'July';
-
-      case 8:
-        return 'August';
-
-      case 9:
-        return 'September';
-
-      case 10:
-        return 'October';
-
-      case 11:
-        return 'November';
+import { useToastError } from '@/components';
 
-      case 12:
-        return 'December';
-    }
+import { useCategoriesList } from './categories.service';
+import { useMoviesList } from './movies.service';
+import { Category } from './movies.types';
+
+export const MovieBanner = ({ movieId }: { movieId: undefined | number }) => {
```

Comment:
Il ne faut pas faire comme ca le typage, pour rendre optionnel il faut faire 
```suggestion
export const MovieBanner = ({ movieId }: { movieId?: number }) => {
```